### ISSUE TITLE
long file paths causing a crash

### ISSUE THREAD

**Issue by stavsap** [diStyApps/Safe-and-Stable-Ckpt2Safetensors-Conversion-Tool-GUI#26]
i experienced crashes with long file names, i dont know if there is a limit on the total path length but after debugging it seems that 

`selected_path = file_explorer.SelectedFileSystem.path `

is not the correct full path.

**Comment by diStyApps** [comment]
Yes, there is a limit above 170 characters or something that I can't remember. Also, it's best to avoid long file names.
